AliAnalysisTaskEx01.cxx

```diff
@@ -0,0 +1,265 @@
+/**************************************************************************
+ * Copyright(c) 1998-1999, ALICE Experiment at CERN, All rights reserved. *
+ *                                                                        *
+ * Author: The ALICE Off-line Project.                                    *
+ * Contributors are mentioned in the code where appropriate.              *
+ *                                                                        *
+ * Permission to use, copy, modify and distribute this software and its   *
+ * documentation strictly for non-commercial purposes is hereby granted   *
+ * without fee, provided that the above copyright notice appears in all   *
+ * copies and that both the copyright notice and this permission notice   *
+ * appear in the supporting documentation. The authors make no claims     *
+ * about the suitability of this software for any purpose. It is          *
+ * provided "as is" without express or implied warranty.                  *
+ **************************************************************************/
+
+/* $Id$ */
+
+/* AliAnalysisTaskEx01.cxx
+ *
+ * Template task producing a P_t spectrum and pseudorapidity distribution.
+ * Includes explanations of physics and primary track selections
+ *
+ * Instructions for adding histograms can be found below, starting with NEW HISTO
+ *
+ * Based on tutorial example from offline pages
+ * Edited by Arvinder Palaha
+ */
+#include "AliAnalysisTaskEx01.h"
+
+#include "Riostream.h"
+#include "TChain.h"
+#include "TTree.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TCanvas.h"
+#include "TList.h"
+
+#include "AliAnalysisTaskSE.h"
+#include "AliAnalysisManager.h"
+#include "AliStack.h"
+#include "AliESDtrackCuts.h"
+#include "AliESDEvent.h"
+#include "AliESDInputHandler.h"
+#include "AliAODEvent.h"
+#include "AliMCEvent.h"
+
+ClassImp(AliAnalysisTaskEx01)
+
+//________________________________________________________________________
+AliAnalysisTaskEx01::AliAnalysisTaskEx01() // All data members should be initialised here
+   :AliAnalysisTaskSE(),
+    fOutput(0),
+    fTrackCuts(0),
+    fHistPt(0), 
```

Comment:
Please remove all trailing whitespace in all files, there are 49 occurrences:

```console
$ git log master.. -p | grep -c ' $'
49
```

Reply:
There's still 1 more space in `master` that I cannot find, although on my local branch there is none:
```
git log new-alice-example.. -p | grep -c ' $'
0
```